src/mavlink-router/endpoint.cpp

```diff
@@ -342,7 +342,7 @@ bool Endpoint::accept_msg(int target_sysid, int target_compid, uint8_t src_sysid
         return true;
 
     // This endpoint has the target of message (sysid, but compid is broadcast): accept
-    if (has_sys_id(target_sysid))
+    if ((target_compid == 0 || target_compid == -1) && has_sys_id(target_sysid))
```

Comment:
You need to update the comment above: "but compid is broadcast or non-existent

Reply:
frankly, you guys are weird ... it exactly follows the recommendation given somewhere else to follow style and conventions, and here it exactly follows the convention set in L.336 just slightly in the above ...
and indeed, =0 and not existing have the same effect and could and should be both called "broadcast"
anyway, I guess you were kindly asking me to then also change L.336, so, I'll do this ;)

btw, I'm wondering if one shouldn't better use (target_sysid <= 0) and (target_compid <= 0) in L.337 and L.345, less clumsy and the compiler might also do less code, I would like it better. What do you say?

Reply:
May I ask you to stop with sentences like "frankly, you guys are weird"? It doesn't help in getting things merged/fixed.

Reply:
May I ask you to stop commanding folks when they have perfectly blended in and followed what was some elses mistake. It doesn't help getting things fixed. Politeness is a mutual thing, not unidirectional.

Reply:
@olliw42 it was a simple code review as usual, nothing else. If you disagree, you are perfectly entitled to say "but not having compid is the same as being a broadcast". Perhaps being too succinct in my comment gave the impression I was commanding something, sorry.